src/js/interface.js

```diff
@@ -869,9 +869,17 @@ window.addEventListener("load", function() {
             if (interval)
                 window.clearInterval(interval);
 
+            var realDelay = delay;
+            var itersPerLoop = 1;
+            if (realDelay < 10) {
+                realDelay = 10;
```

Comment:
What is this `realDelay` variable used for?

I don't see it used anywhere else, unless its used to replace the `delay` variable inside the `setInterval` function

Reply:
Of course that was the plan... Doesn't make any difference though because JS seems to assume a minimum of 10 ms anyway. I've fixed this (see commit below). I've also added a fix for a travis ci problem.